site/pmguides/src/google-oauth-in-postman/google-oauth-in-postman.md

```diff
@@ -0,0 +1,192 @@
+
+![Services Library](assets/services_library.png)
+
+Search for “Google Sheets API” and enable it. While you can make use of any other API and still have the same flow documented in this blog post, the Google sheets API will be used for our Authorization example.
+
+![Enable Sheets](assets/enable-sheets.png)
+
+
+<!-- ------------------------ -->
+## Get your credentials
+Duration: 3
+
+When you’re done enabling the API you intend to work with, click the Create Credentials button at the top right of the API dashboard.
+
+![Create Credentials](assets/create-credentials.png)
+
+Under the **Credentials Type** section, the Google sheets API should be auto-selected, but you can change this to any other API you intend to work with. Select **user data** as the data you want to access and click next.
+
+![Credentia's Type](assets/credentials-type.png)
+
+Give your app a name, select a support email address, and upload a logo for your app if you have one. Save and continue. 
+
+Click the **Add or remove scopes** button, and search for “sheets.” Select the read-only option and click the update button.
+
+![Scopes](assets/select-scopes.png)
+
+Click **Save and continue**. In the OAuth Client ID, select **Web Application**  as your application type and give the client a name. 
+
+![Oauth Client ID](assets/oauth-client-id.png)
+
+In the **Authorized redirect URIs**, click the **Add URI** button. The URI you add will depend on what Postman environment you’re testing from. If you’re testing from Postman's web application, add the redirect URI https://oauth.pstmn.io/v1/browser-callback.
+
+If you’re testing from the Postman Desktop, add the URI https://oauth.pstmn.io/v1/callback. If you switch between the web and desktop application during your development workflow, you should add both URIs as redirect URI. 
+
+![Authorized redirect URIs](assets/authorized-redirect-uri.png)
+
+
+Copy your client ID and ensure that you click the download button to download all your credentials.
+
+![Download Your Credentials](assets/download-your-credentials.png)
+
+When your credentials have been successfully downloaded, navigate to the credentials tab and click on the name of the newly added credential.
+
+![Sheets API Credentials](assets/sheets-credentials.png)
+
+You will see your client credentials(client ID and client secret) on the right-hand side of your dashboard.
+
+![Credentials](assets/credentials.png)
+
+<!-- ------------------------ -->
+## Authorize with Postman
+Duration: 5
+
+Now that you have your client ID and client secret, you can use Postman in your automated API testing or API development workflow to authorize Google APIs. 
+
+Once you have your collection created in Postman and you’ve created a new request. Navigate to https://sheets.new, this will automatically create a new Google sheet on your Google account. Populate this sheet with random data and give it a random name. 
```

Comment:
I would break up the sentences a little differently. For example:

Once you have your collection created in Postman and you’ve created a new request, navigate to https://sheets.new. This will automatically create a new Google sheet on your Google account.